WORKSPACE

```diff
@@ -29,3 +29,16 @@ googletest_repositories()
 googleapis_repositories()
 mixerapi_repositories()
 
+git_repository(
+    name = "io_bazel_rules_go",
+    commit = "7991b6353e468ba5e8403af382241d9ce031e571",  # Aug 1, 2017 (gazelle fixes)
+    remote = "https://github.com/bazelbuild/rules_go.git",
+)
+
+load("@io_bazel_rules_go//go:def.bzl", "go_repositories", "go_repository")
+
+go_repositories()
+
+load("@mixerapi_git//:api.bzl", "go_istio_api_dependencies")
+
+go_istio_api_dependencies()
```

Comment:
Why do I need this go stuff in my c++  library?

Does that mean I need to add this in Istio/Proxy WORKSPACE?

Reply:
Yes, but you already import go dependencies in istio/proxy. The API repo is optimized for golang clients as we want third party adaptors to use it.